composer.json

```diff
@@ -215,6 +216,10 @@
             "type": "git",
             "url": "https://github.com/open-y-subprojects/openy_docs"
         },
+        {
+            "type": "vcs",
+            "url": "https://github.com/YCloudYUSA/useless_machines"
+        },
```

Comment:
```suggestion
```

Reply:
jk this [won't work anyway](https://getcomposer.org/doc/05-repositories.md#repository) since it's not in the root package.